include/APM/ApmDevice.hpp

```diff
@@ -0,0 +1,97 @@
+/**
+ * Class to represent the ApmDevice Object.  Holds references
+ * to many of the key objects used by this device
+ */
+
+#ifndef APM_APMDEVICE_HPP
+#define APM_APMDEVICE_HPP
+
+#include <EVT/io/pin.hpp>
+#include <EVT/io/UART.hpp>
+#include "ApmUart.hpp"
+
+namespace APM {
+
+namespace IO = EVT::core::IO;
+
+enum class ApmMode {
+    OFF = 0u,
+    ACCESSORY = 1u,
+    ON = 2u
+};
+
+class ApmDevice {
+public:
+    static constexpr IO::Pin ACCESSORY_SW = IO::Pin::PA_5;
+    static constexpr IO::Pin CHARGE_SW = IO::Pin::PA_6;
+    static constexpr IO::Pin VICOR_SW = IO::Pin::PA_4;
+    static constexpr IO::Pin KEY_ON_UC = IO::Pin::PB_5;
+
+    /**
+     * Create a new ApmDevice
+     * Initializes the IO Devices
+     * @param baud the baudrate for the UART device
+     */
+    explicit ApmDevice(IO::GPIO &accessorySwGpio, ApmUart &apmUart, IO::GPIO &chargeSwGpio,
+                       IO::GPIO &keyOnSwGpio, IO::GPIO &vicorSwGpio);
+
+    /**
+     * Returns the ApmUart object by reference
+     * @return const reference to the ApmUart object
+     */
+    [[nodiscard]] ApmUart& getApmUart() const;
```

Comment:
All caps on UART

Reply:
If the class name is updated to `APMUart` then I don't think it makes sense for this method to be `getApmUART`.  It feels inconsistent.

Reply:
Yeah I would agree, that can be left as is